solidity/contracts/converter/LiquidTokenConverter.sol

```diff
@@ -99,7 +99,7 @@ contract LiquidTokenConverter is BancorConverter {
             return saleRate(_amount);
 
         // invalid input
-        revert();
+        revert("BANCOR_ERR_INVALID_TOKEN");

```

Comment:
@yudilevi : See previous comment (in `BancorFormula.sol`) about revert messages in read-only functions.